content/en/hub/initial-setup/storage/encryption.md

```diff
@@ -79,8 +79,14 @@ Must be longer than 8 characters
 
 ## Locking and Unlocking Datasets
 
+Status of a dataset can be determined based on the icon used after the name.  
+
+Dataset Unlocked Icon : <mat-icon _ngcontent-iee-c469="" role="img" fontset="mdi-set" fonticon="mdi-lock-open-variant" class="mat-icon notranslate mdi-set mdi-lock-open-variant mat-icon-no-color ng-star-inserted" aria-hidden="true"></mat-icon>
+
+Dataset Locked Icon : <mat-icon _ngcontent-iee-c469="" role="img" fontset="mdi-set" fonticon="mdi-lock" class="mat-icon notranslate mdi-set mdi-lock mat-icon-no-color ng-star-inserted" aria-hidden="true"></mat-icon>
```

Comment:
These icons aren't showing up in the site preview. Try reworking to use this syntax instead: `<i class="material-icons" aria-hidden="true" title="<hovertext>"><iconname></i>`

Reply:
Still not working.  This is the exact icon that the UI is using.  Are they using more than one Material Design Icon set?